### ISSUE TITLE
Transformer function missing in HGTAN/models.py

### ISSUE THREAD

**Issue by amulyalakku** [lixiaojieff/HGTAN#2]
In the training/test_Transformer.py file, there is a function call to the Transformer function, but there is no Transformer function in HGTAN/models.py or even in the complete solution.

model = Transformer(
        args.length,
        rnn_unit=args.rnn_unit,
        n_hid=args.hidden,
        tgt_emb_prj_weight_sharing=args.proj_share_weight,
        d_k=args.d_k,
        d_v=args.d_v,
        d_model=args.d_model,
        d_word_vec=args.d_word_vec,
        n_head=args.n_head,
        dropout=args.dropout).to(device)

**Comment by lixiaojieff** [comment]
Thank you for your question, there are some mistakes here, the model name should be 'HGTAN', and corresponding modifications have been made in the code!

**Comment by amulyalakku** [comment]
Seeing the below error after using the updated code. Please look into this.
<img width="1412" alt="Screen Shot 2021-10-28 at 8 15 43 PM" src="https://user-images.githubusercontent.com/17248698/139357322-d193cb2e-6efb-409f-a214-6bbd4ba27d42.png">

**Comment by lixiaojieff** [comment]
What about now?

**Comment by amulyalakku** [comment]
Sorry for the trouble. Still seeing the error with the code.
<img width="1421" alt="Screen Shot 2021-10-28 at 8 22 44 PM" src="https://user-images.githubusercontent.com/17248698/139357792-d98734b9-950d-472c-b4fe-1b4ec6f037f1.png">

**Comment by lixiaojieff** [comment]
Deeply sorry!  There was something wrong with the file when I uploaded the code and it has been corrected.

**Comment by amulyalakku** [comment]
No worries. Thank you for your quick response. Our team really appreciate it. Do you have any idea about the below issue?
<img width="1436" alt="Screen Shot 2021-10-28 at 8 45 15 PM" src="https://user-images.githubusercontent.com/17248698/139359645-b2b76c9a-5398-4e38-b2aa-268e1bc89ee6.png">

**Comment by lixiaojieff** [comment]
It may be that your Pytorch and CUDA versions are incompatible?

**Comment by amulyalakku** [comment]
Ok, Thank you very much for your support. I will crosscheck with the versions and try to compile the code. Also, Thank you to your whole team for your amazing work.
